pkg/dependencies/version.go

```diff
@@ -181,6 +184,32 @@ func gradleVersion(dep *Dependency, projectDir string) (*semver.Version, error)
 	return version, nil
 }
 
+func JazzerVersion(classPath string) (*semver.Version, error) {
+	return extractVersion(classPath, jazzerRegex, "jazzer")
+}
+
+func JUnitVersion(classPath string) (*semver.Version, error) {
+	return extractVersion(classPath, junitRegex, "junit-jupiter-engine")
+}
+
+func JazzerJSVersion() (*semver.Version, error) {
+	cmd := exec.Command("npx", []string{"jazzer", "--version"}...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	return extractVersion(string(output), nodeRegex, "jazzer.js")
+}
+
+func JestVersion() (*semver.Version, error) {
+	cmd := exec.Command("npx", []string{"jest", "--version"}...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	return extractVersion(string(output), nodeRegex, "jest")
+}
```

Comment:
By using `cmd.Output` instead of `cmd.CombinedOutput` and wrapping the error in `cmdutils.ExecError`, the stderr of the command is printed on failure, which allows to debug failures.

```suggestion
func JazzerJSVersion() (*semver.Version, error) {
	cmd := exec.Command("npx", []string{"jazzer", "--version"}...)
	output, err := cmd.Output()
	if err != nil {
		return nil, cmdutils.WrapExecError(errors.WithStack(err), cmd)
	}
	return extractVersion(string(output), nodeRegex, "jazzer.js")
}

func JestVersion() (*semver.Version, error) {
	cmd := exec.Command("npx", []string{"jest", "--version"}...)
	output, err := cmd.Output()
	if err != nil {
		return nil, cmdutils.WrapExecError(errors.WithStack(err), cmd)
	}
	return extractVersion(string(output), nodeRegex, "jest")
}
```